### ISSUE TITLE
web: Hiding a Ruffle instance results in WebGL warnings and panic

### ISSUE THREAD

**Issue by Toad06** [ruffle-rs/ruffle#1264]
Setting the style `display: none;` to a Ruffle instance produces the following WebGL warnings:

Chrome:
```
[.WebGL-000001C65BA3EDF0] GL_INVALID_FRAMEBUFFER_OPERATION: Framebuffer is incomplete.
[.WebGL-000001C65BA3EDF0] GL_INVALID_FRAMEBUFFER_OPERATION: Draw framebuffer is incomplete
WebGL: too many errors, no more errors will be reported to the console for this context.
```

Firefox:
```
WebGL warning: clear: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0: Attachment has no width or height.
WebGL warning: clear: Framebuffer must be complete.
WebGL warning: drawElementsInstanced: Framebuffer not complete. (status: 0x8cd6) COLOR_ATTACHMENT0: Attachment has no width or height.
(...)
WebGL warning: drawElementsInstanced: TEXTURE_2D at unit 0 is incomplete: The dimensions of `level_base` are not all positive.
(...)
WebGL: No further warnings will be reported for this WebGL context. (already reported 32 warnings)
```

Then changing the style back to `display: inline-block;` crashes Ruffle:
`panicked at 'called Result::unwrap() on an Err value: "WebGL: Error in renderbuffer_storage_multisample (color): 1286"', render\webgl\src\lib.rs:736:35`

Sample:
[displayTest.zip](https://github.com/ruffle-rs/ruffle/files/5333679/displayTest.zip)

**Comment by Toad06** [comment]
Fixed by #1767.

**Comment by Toad06** [comment]
Re-opening as this regressed. This no longer hits a panic but seriously affects the rendering and still logs warnings in the console.

From testing (see the sample in the first post), it appears that the regression was introduced in https://github.com/ruffle-rs/ruffle/commit/e45810f585b634b6f1e374e406ccecde1c16ead6.

@relrelb , pinging to let you know. 😉
